src/chrome/chromeConnection.ts

```diff
@@ -155,7 +155,7 @@ export class ChromeConnection {
                 if (Array.isArray(responseArray)) {
                     // Filter out extension targets and other things
                     // Non-chrome scenarios don't always specify a type, so filter to include ones without a type at all
-                    let pages = responseArray.filter(target => target && (!target.type || target.type === 'page'));
+                    let pages = responseArray;//.filter(target => target && (!target.type || target.type === 'page'));
```

Comment:
In case of nw.js, we can attach to both, its the developer call on which context he wants to debug. Should I add `background_page` and `app` as a target?